Source/WebInspectorUI/UserInterface/Base/Main.js

```diff
@@ -1413,8 +1410,7 @@ WI.tabContentViewClassForRepresentedObject = function(representedObject)
 
     if (representedObject instanceof WI.DOMStorageObject || representedObject instanceof WI.CookieStorageObject ||
         representedObject instanceof WI.DatabaseTableObject || representedObject instanceof WI.DatabaseObject ||
-        representedObject instanceof WI.ApplicationCacheFrame || representedObject instanceof WI.IndexedDatabaseObjectStore ||
```

Comment:
there's a few other values on `WI.ContentViewCookieType` that probably should be removed